packages/@modelfusion-llamacpp-bindings/src/LlamaCppBindingsCompletionModel.ts

```diff
@@ -0,0 +1,253 @@
+import {
+  TextGenerationBaseModel,
+  TextGenerationModel,
+  TextGenerationModelSettings,
+  TextGenerationPromptTemplateProvider,
+  InstructionPrompt,
+  ChatPrompt,
+  BasicTokenizer,
+  FullTokenizer,
+  TextGenerationResult,
+} from "modelfusion";
+import { AbstractModel } from "modelfusion/model-function/AbstractModel";
```

Comment:
Note sure if this important will work in various environments (once it's compiled). `AbstractModel` is exported in `modelfusion/internal` so you should be able to use `import { AbstractModel } from "modelfusion/internal";` instead.